lighthouse-cli/test/smokehouse/test-definitions/pwa/pwa3-expectations.js

```diff
@@ -16,8 +16,8 @@ const pwaRocksExpectations = {...pwaDetailsExpectations, hasIconsAtLeast512px: f
 module.exports = [
   {
     lhr: {
-      requestedUrl: 'https://pwa.rocks',
-      finalUrl: 'https://pwa.rocks/',
+      requestedUrl: 'https://connorjclark.github.io/pwa.rocks/',
```

Comment:
maybe add a comment for future code archaeologists that this is the archived version of `https://pwa.rocks`, see repo at `https://github.com/pwarocks/pwa.rocks` or something